### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by marco-souza**
- Whatch .js files
- Type do storage properties
- Fix Text component
- Flow ignore compiled code
- Add content to docs page

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -1 +1,4 @@
-test: "ola maluco doido"
\ No newline at end of file
+test: "ola maluco doido"
```
Comment (danilo-p) [inline config/default.yml]: Check if it is used. If it is just an unused test, just remove it.

File: docs/src/app/main/index.js [hunk 2]
```diff
@@ -16,7 +17,8 @@ class Component extends React.Component<Props> {
     render() {
         return (
             <h5>
-                <div>Test</div>
+                <Text>Test Text</Text>
```
Comment (danilo-p) [inline docs/src/app/main/index.js]: Why this `docs/src/app/main/index.js` exists? It is not imported on the `docs/src/app/index.js`. If it is not used, please remove or fix the usage of this module.

Comment (marco-souza) [reply docs/src/app/main/index.js]: I'll remove the main folder.

File: config/default.yml (hunk 1 above)
Comment (marco-souza) [reply config/default.yml]: Fixed

**Pull request merged**